packages/typir/src/kinds/function/function-kind.ts

```diff
@@ -96,6 +96,12 @@ export type InferFunctionCall<T = unknown> = {
  */
 
 
+export interface FunctionPredefinedService {
```

Comment:
Should this stay "predifined"? The others are named factory

Reply:
Yes, I missed that, thanks!